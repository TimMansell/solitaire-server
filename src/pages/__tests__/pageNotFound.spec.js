import { shallowMount } from '@vue/test-utils';
import PageNotFound from '@/pages/PageNotFound.vue';

describe('PageNotFound.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(PageNotFound);

    expect(wrapper).toMatchSnapshot();
  });
});
