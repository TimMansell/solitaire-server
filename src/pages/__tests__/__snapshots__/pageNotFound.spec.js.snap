// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound.vue matches snapshot 1`] = `
<gameoverlay-stub centercontent="true" showlogo="true" data-test="404"> Page Not Found :(<p>Click the button below to be taken back to the game</p>
  <homebutton-stub></homebutton-stub>
</gameoverlay-stub>
`;
